packages/paypal-js/types/script-options.d.ts

```diff
@@ -45,4 +45,5 @@ export interface PayPalScriptOptions
         PayPalScriptDataAttributes,
         ScriptAttributes {
     sdkBaseUrl?: string;
+    environment?: "production" | "sandbox";
```

Comment:
🔤 There ought to be a linting rule that does this for us. 😁 
```suggestion
    environment?: "production" | "sandbox";
    sdkBaseUrl?: string;
```